profab/role/ami/ubuntu.py

```diff
@@ -16,7 +16,7 @@ class Configure(Role):
     def ami(self, region, bits, size):
         """Return the AMI that was passed in to the role.
         """
-        disk = 'ebs' if size == 't1.micro' else 'instance'
```

Comment:
That's not really an argument for why EBS being the default is the right thing to do. If the user wants an instance disk how do they do that?